toolkit/offchain/src/permissioned_candidates.rs

```diff
@@ -153,14 +155,14 @@ mod tests {
 		let total_collateral = body.total_collateral().unwrap();
 		assert_eq!(
 			collateral_return.amount().coin().checked_add(&total_collateral).unwrap(),
-			collateral_utxo().value.lovelace.into()
+			greater_payment_utxo().value.lovelace.into()
 		);
 	}
 
 	#[test]
-	fn update_d_param_tx_regression_test() {
+	fn update_permissioned_candidates_tx_regression_test() {
 		// We know the expected values were obtained with the correct code
-		let script_utxo_lovelace = 1060260;
+		let script_utxo_lovelace = 1952430;
```

Comment:
Another C-P error, this value is the real one with test permissioned candidates datum. It wasn't critical, but now the test is closer to the reality.